From 9ba8adbfa45eb1800fe65c49836a88eb9ce40832 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 09 May 2023 15:31:57 +0300
Subject: [PATCH] [SW-143953]  i2c_direct: remove legacy command codes

command codes 135-209 are defined as reserved in the NVME spec, but
there is a lot of unused legacy code implementing all sort of data
for this range. this code is being removed.

Change-Id: I061cd9dd4ca64cc787ba4148107e9dd89c1ff1d4
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/lib/preboot/module_i2c_slave.c b/lib/preboot/module_i2c_slave.c
index ef1f03c..aef0a3d 100644
--- a/lib/preboot/module_i2c_slave.c
+++ b/lib/preboot/module_i2c_slave.c
@@ -72,12 +72,6 @@
 	case HL_ASIC_TEMP_MAX_VAL:
 		ret = get_composite_temp();
 		break;
-	case HL_DEVICE_STATUS_CMD_CODE:
-		ret = HL_DEVICE_STATUS_CMD_CODE_LEN;
-		break;
-	case HL_DEVICE_STATUS:
-		ret = HL_DEVICE_STATUS_PREBOOT;
-		break;
 	case HL_PCIE_VENDOR_ID_VAL_LOW:
 	case HL_PCIE_VENDOR_ID_VAL_HIGH:
 	case HL_PCIE_DEVICE_ID_VAL_LOW:
